PACKAGES.md

```diff
@@ -0,0 +1,70 @@
+# Plone frontend packages
+
+## `@plone/types`
+
+Plone types is worth to have its own entry, as it's an special development package.
+It contains the Plone typings for TypeScript.
+It's considered a core package, and it's the only package that the other core packages can rely on (as devDependency).
+
+This package contains `.d.ts` typing definitions, curated by hand.
+Due to the nature of this package, it does not need bundling.
+It's publised "as it is", so you can import the type definitions from anywhere in your code.
+
+## Core packages
+
+- `@plone/registry`
+- `@plone/client`
+- `@plone/components`
+
+### Rules
+
+They can't depend on any other `@plone/*` package, with only one exception: `@plone/types`.
+They must be published and bundled in a traditional (transpiled) way.
+The bundle of these packages must work on both commonjs and ESM environments.
```

Comment:
What is ESM? "European Stability Mechanism"? The first time an acronym is used, you must spell out its proper name, and optionally follow it with its acronym in parentheses.

European Stability Mechanism (ESM)

Reply:
ECMAScript Module (ESM):

https://medium.com/@halilatilla/differences-between-javascript-modules-cjs-amd-umd-and-esm-f60124de131b#:~:text=ECMAScript%20Module%20(ESM)%3A,static%20analysis%20and%20tree%20shaking.